docs/modules/clusters/pages/ucn-data-structures.adoc

```diff
@@ -0,0 +1,80 @@
+= Data Structures
+:description: To allow your data structures to resolve your user code, you must reference the required namespace in the data structure configuration. You can configure the same namespace for multiple data structures.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+Any data structure without a specific {ucn} definition in its configuration uses the `default` namespace, if it has been configured. For further information on defining a `default` namespace, see the xref:clusters:ucn-non-associated.adoc[] topic.
+
+WARNING: If your data structure configuration specifies a namespace that has not been configured, a fatal error is returned.
```

Comment:
This was actually a point of discussion with Neil while playing around with UCN - the fatal error is only encountered when a data structure which references a non-existent Namespace is **operated upon**, there is no errors thrown during startup as you might see elsewhere in Hazelcast (fail-fast configuration checking).

To further demonstrate, here's an example:
1.  Configure `my_map` as an `IMap` that references `bad_namespace` as it's `user-code-namespace`, but do not create any namespaces in the config
2. Start the Hazelcast member with this config, you will observe no errors
3. Make a call to the `my_map` `IMap` (whether to read, write, etc), and an error which prevents the operation will be encountered